src/lib/add-on/clients/add-on_auto.rb

```diff
@@ -1,328 +1,319 @@
-# encoding: utf-8
-
-# File:
-#      add-on_auto.ycp
-#
-# Module:
-#      Add-On autoinstallation and configuration
-#
-# Summary:
-#      Add-On autoinstallation preparation
-#
-# Authors:
-#      Jiri Srain <jsrain@suse.cz>
-#
-# $Id$
-#
-module Yast
-  class AddOnAutoClient < Client
-    def main
-      Yast.import "Pkg"
-      Yast.import "UI"
-      textdomain "add-on"
-
-      Builtins.y2milestone("----------------------------------------")
-      Builtins.y2milestone("add-on auto started")
-
-      Yast.import "AddOnProduct"
-      Yast.import "Progress"
-      Yast.import "AutoinstSoftware"
-      Yast.import "PackageCallbacks"
-      Yast.import "Label"
-      Yast.import "AutoinstGeneral"
-      Yast.import "PackageLock"
-      Yast.import "Installation"
-      Yast.import "String"
+# rubocop:disable Naming/FileName
```

Comment:
Um, I cannot see any `.rubocop.yml` file in the project. Without it tweaking the Rubocop config is probably useless...

And I'd prefer disabling it in the Rubocop config file instead of polluting the source code. See an example [here](https://github.com/yast/yast-registration/blob/master/.rubocop.yml#L34-L36).

Reply:
You're right. I took borrowed the `.rubocop.yml` from `yast-storage-ng` to lint the code in my local copy... and then I forgot to remove those comments :innocent:

> And I'd prefer disabling it in the Rubocop config file instead of polluting the source code. 

Me too :) :+1:

Reply:
Well, you can add it to the package. The long term goal is to use Rubocop for all code. We just did not have time to adapt all packages.

Reply:
Ok, I'll do it in its own PR :)